wp-content/themes/ipbs-largo/less/widgets.less

```diff
@@ -9,4 +9,57 @@
         margin-bottom: @measure * 2;
         color: @blue;
     }
+}
+
+// member stations widget
+.widget.ipbs-member-stations {
+    margin-bottom: @measure * 2;
+    ul {
+        @media (min-width: 1000px) {
```

Comment:
Would it be better to use a `flex` layout with flex-basis on the `li` of about 380px, no flex-shrink, yes flex-grow? That way we have three-column, two-column, and one-column layouts.

🤔 but that makes borders harder.